### DISCUSSION TITLE
How can I prevent CheckPicker from rendering selected items?

### DISCUSSION THREAD

**Discussion by ffersants** [rsuite/rsuite#2028] [Q&A]
Hi guys! I'd like some help... I'm trying to prevent rendering the marked items in the CheckPicker component, can somebody help me? My goal is to render, in another part of the page, the Tag component with the selected value

**Comment by SevenOutman** [comment]
Hi Fernando, I think you can do this by removing selected items from the `data` passed to `<CheckPicker>`.

```jsx
// pseudo code
const unselectedItems = without(allItems, selectedItems);

return (
  <ChechPicker
    data={unselectedItems}
    onChange={value => add(selectedItems, value)}
  />
);
```
